### PR TITLE
Add `@mavthedev/svodals`

### PR DESCRIPTION
**Pull request by mavdotj**

### CODE REVIEW

**Comment by mavdotj** [comment]
uhh

**Comment by mavdotj** [comment]
...

**Comment by MacFJA** [comment]
Is it a tool or a component ?

From what I read from your code/example it look more like a new component than a development/building tool.

**Comment by mavdotj** [comment]
im kind of indecisive on if its a component or tool since it is a mix of both. I'll put component tho

**Comment by mavdotj** [comment]
wait a minute... didn't you make one of the modules i use?

Yup, svelte persistent store

File: src/routes/components/components.json [hunk 1]
```diff
@@ -2626,5 +2626,15 @@
...
+	{
+		"title": "svodals",
+		"url": "https://github.com/mavthedev/svodals",
+		"description": "Cybernatically enhanced modals",
+		"npm": "@mavthedev/svodals",
+		"addedOn": "2022-07-07",
+		"category": "User Interaction",
+		"tags": ["modals"],
+		"stars": 0
+	}
```
Comment (MacFJA) [inline src/routes/components/components.json]: There is a tabulation at the end of the line
```suggestion
	}
```

**Review by MacFJA — CHANGES_REQUESTED** [review]

**Review by lachlancollins — APPROVED** [review]

**Comment by mavdotj** [comment]
does this package even still work lol

**Pull request merged**